server/src/manager/manager.module.ts

```diff
@@ -8,5 +8,6 @@ import { ManagerService } from './manager.service';
   imports: [TypeOrmModule.forFeature([Manager])],
   controllers: [ManagerController],
   providers: [ManagerService],
+  exports: [ManagerModule],
```

Comment:
```suggestion
  exports: [ManagerService],
```

exporter le module est inutile et il manque l'export du service